webpack/src/Extends/Host/MonitoringTab/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { Table, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { STATUS } from 'foremanReact/constants';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { get } from 'foremanReact/redux/API';
+import { selectAPIStatus, selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import Loading from 'foremanReact/components/Loading';
+import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
+import DefaultLoaderEmptyState from 'foremanReact/components/HostDetails/DetailsCard/DefaultLoaderEmptyState';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+
+const STATUS_ICONS = {
+  'ok': 'pficon-ok status-ok',
+  'warning': 'pficon-info status-warn',
+  'critical': 'pficon-error-circle-o status-error',
+}
```

Comment:
instead of using classnames, can you use the icons from PF4?

```js
import {
  ExclamationTriangleIcon,
  CheckCircleIcon,
  ExclamationCircleIcon,
} from '@patternfly/react-icons';
```

```js
const STATUS_ICONS = {
  ok: CheckCircleIcon,
  warning: ExclamationTriangleIcon,
  critical: ExclamationCircleIcon,
}
``` 

later I believe you can use it this way:

```js
<Td>{STATUS_ICONS[result.status]} {result.status_label}</Td>
```

Reply:
That is much nicer. My university programming professor would have yelled at me for using hardcoded strings instead of constants.

Reply:
This worked for me:
```javascript
const STATUS_ICONS = {
  'ok': CheckCircleIcon,
  'warning': ExclamationTriangleIcon,
  'critical': ExclamationCircleIcon,
};

const STATUS_STYLES = {
  'ok': 'ok',
  'warning': 'warn',
  'critical': 'critical',
};

const status_icon = (result_status) => {
  const cls = STATUS_ICONS[result_status] || QuestionCircleIcon;
  const style = STATUS_STYLES[result_status] || 'question';
  return createElement(cls, { className: `status-${style}` });
};
```

Reply:
looks good,
do you still need to use the style?

if you do, there's an option to pass the `className` into the Icon component, so no need for the `createElement`